### PR TITLE
github: Rename master to main

### PR DESCRIPTION
**Pull request by juan-lunarg**

### CODE REVIEW

**Comment by Tony-LunarG** [comment]
I thought we were only doing this in Khronos repos

**Comment by juan-lunarg** [comment]
> I thought we were only doing this in Khronos repos

Agreed there isn't a need to do it here. It's just about consistency. Most of the people who contribute here work on Khronos repos where the default will be main.

**Comment by juan-lunarg** [comment]
To be clear I don't think any other LunarG repo needs to change the default branch. Just this one.

**Review by charles-lunarg — COMMENTED** [review]
I don't see any mention of the master->main switch in the readme or build. (I remember seeing a mention of that in the Vulkan-Tools repo)

**Review by christophe-lunarg — APPROVED** [review]
Consistency sounds good to me

**Comment by Tony-LunarG** [comment]
Ok

**Comment by juan-lunarg** [comment]
> I don't see any mention of the master->main switch in the readme or build. (I remember seeing a mention of that in the Vulkan-Tools repo)

Updated README to include that

> Consistency sounds good to me

Currently there are only 2 active PUBLIC repos in LunarG

image

`Vulkan-Tools` and `gfxreconstruct`

reconstruct uses the `dev` convention which is fine. But seeing as `Vulkan-Tools` devs often work on `Khronos` repos, it ultimately makes more sense to use `main` for consistency.

**Review by charles-lunarg — APPROVED** [review]
Thanks!

File: vkconfig/CHANGELOG.md [hunk 1]
```diff
@@ -107,7 +107,7 @@
 - Add build date in the title bar
 
 ### Improvements:
-- Add link to [Vulkan Guide layers](https://github.com/KhronosGroup/Vulkan-Guide/blob/master/chapters/development_tools.md#vulkan-layers) list within the help menu
+- Add link to [Vulkan Guide layers](https://github.com/KhronosGroup/Vulkan-Guide/blob/main/chapters/development_tools.md#vulkan-layers) list within the help menu
```
Comment (juan-lunarg) [inline vkconfig/CHANGELOG.md:110]: Also using `main` consistently helps with minor issues like this in the documentation.

**Pull request merged**